data/address-geocoders-locators/index.html

```diff
@@ -52,89 +59,32 @@
-UGRC also provide pre-built clients to use the Web API that can be downloaded below. UGRC hopes to have the data and tools
-ready for your needs to geocode." %}
+webapp="https://api.mapserv.utah.gov" abstract="Adding a geographic point for an address is a fundamental GIS task. To handle this task, UGRC has created many services and tools to cater to the wide geocoding user base. Starting from the ground up, UGRC helps maintain statewide address points and roads datasets. These two datasets are the foundation for geocoding. Building on top of the reference data, UGRC creates and maintains ArcGIS Locators. Locators are where most of the magic happens - taking the input data and finding a location from the reference data. These locators are available for download on this page. With the reference data and the locators blocks in place, UGRC can put together geocoding systems and tools. The UGRC WebAPI is a web service that can geocode over the web via http. Users can send requests via their favorite programming language, application, or web browser. This service has intimate Utah addressing knowledge to get the best results out of user input and is the recommended way to geocode addresses. UGRC also provides pre-built clients to use the Web API that can be downloaded below. UGRC hopes to have the data and tools ready for your needs to geocode." %}
 <div class="grid package">
   <div class="grid__col grid__col--12-of-12">
-    <h3 id="WebAPI">UGRC Web API</h3>
+    <h3 id="OfficialClient">Official UGRC API Client</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <p>
-      {% include fa_icon.html api=true class="svg-inline--fa fa-w-20 fa-4x
-      fa-pull-left" %} The UGRC Web API is an http enabeled service for accessing (via the internet) the geospatial data that we store in the State Geographic Information Database (SGID). The endpoints provided by the API are a great way to add geospatial functionality to your web pages and applications or desktop workflows. You will be able to geocode addresses and search all the data in the SGID. To get started,
-      <a href="https://developer.mapserv.utah.gov/StartupGuide"
-        >read the getting started guide</a
-      >. There are also examples of geocoding in
-      <a href="https://github.com/agrc/GeocodingSample"
-        >multiple programming languages</a
-      >.
+      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform, stand-alone desktop geocoding tool designed to carefully guide you step-by-step through your geocoding tasks. This app is our most user friendly geocoding client for geocoding <code>*.csv</code> files. It does not require a license or subscription to use and automatically download updates as soon as they are available to stay current and add new features.
```

Comment:
```suggestion
      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform, stand-alone desktop geocoding tool designed to carefully guide you step-by-step through your geocoding tasks. This app is our most user friendly geocoding client for geocoding <code>*.csv</code> files. It does not require a license or subscription to use and automatically downloads updates as soon as they are available to stay current and add new features.
```

I missed an `s` the first time around.